redaxo/src/core/lib/api_function.php

```diff
@@ -90,6 +90,49 @@ public static function factory()
         return null;
     }
 
+    /**
+     * Returns an array containing the `rex-api-call` and `_csrf_token` params.
+     *
+     * The method must be called on sub classes.
+     *
+     * @return array
+     */
+    public static function getUrlParams()
```

Comment:
In der `rex_csrf_token`-Klasse haben wir `getHiddenField` und `appendToUrl`.
Hier haben wird `getHiddenFields` und `getUrlParams`.
Wollen wir das vereinheitlichen? Also entweder hier auch `appendToUrl`, oder in der csrf-klasse `getUrlParam`..

Reply:
Was sagste denn hierzu @staabm?

Reply:
ich denke für beides getUrlParams() ist sinnvoll.

appendToUrl() ist irgendwie "in die falsche richtung".